proxy/http/HttpSM.cc

```diff
@@ -5994,6 +5994,9 @@ HttpSM::setup_server_send_request()
 
   milestones[TS_MILESTONE_SERVER_BEGIN_WRITE] = Thread::get_hrtime();
   server_entry->write_vio                     = server_entry->vc->do_io_write(this, hdr_length, buf_start);
+
+  // Make sure the VCs is on the correct timeouts
+  server_session->get_netvc()->set_inactivity_timeout(HRTIME_SECONDS(t_state.txn_conf->transaction_no_activity_timeout_out));
```

Comment:
Are we now setting the inactivity timeout multiple times for POSTs and PUSH?

Reply:
We could move the inactivity_timeout to setup_server_request_api.  I think that would avoid the POST/PUSH case.   Not sure if that would still hit the transform cases.